README.md

```diff
@@ -144,7 +144,11 @@ For more details, see the [update system documentation](sources/updater/).
 
 ### Update methods
 
-There are several ways of updating your Bottlerocket hosts:
+There are several ways of updating your Bottlerocket hosts.
+
+For EKS variants of Bottlerocket, we recommend using the [Bottlerocket update operator](#bottlerocket-update-operator) for automated updates.
```

Comment:
I think linking directly to the brupop project might be more helpful than linking down below, since down below is basically just a link to the project.